src/services/item/plugins/published/plugins/search/factory.ts

```diff
@@ -0,0 +1,19 @@
+import { MeiliSearch } from 'meilisearch';
+import { DataSource } from 'typeorm';
+
+import { BaseLogger } from '../../../../../../logger';
+import { MEILISEARCH_MASTER_KEY, MEILISEARCH_URL } from '../../../../../../utils/config';
+import FileService from '../../../../../file/service';
+import { MeiliSearchWrapper } from './meilisearch';
+
+export const createMeiliSearchWrapper = (
```

Comment:
I made this here only to simplify the registration of the MeilisearchService. Because the MeiliSearchService needs a meilisearch instance. If you think we should instanciate directly in the dependencies file, we also can do that. I don't know if my comment here can help ^^